api/app.go

```diff
@@ -125,27 +117,29 @@ func (a App) healthCheckHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *App) initializeRoutes() {
-	a.Router.HandleFunc("/parcelle/{idu:"+iduRegex+"}", a.getById).Methods("GET")
+	a.Router.PathPrefix("/viewer/").Handler(http.StripPrefix("/viewer/", http.FileServer(http.Dir("./views")))).Methods("GET")
+
+	a.Router.Handle("/parcelle/{idu:"+iduRegex+"}", Use(LogMw).ThenFunc(a.getById)).Methods("GET")
 
-	a.Router.HandleFunc("/parcelle", a.findByPosition).Queries(
+	a.Router.Handle("/parcelle", Use(LogMw).ThenFunc(a.findByPosition)).Queries(
```

Comment:
It's clearer with Use() but this should also be working ? Let's keep Use() anyway for further middlewares
```
	a.Router.Handle("/parcelle", LogMw.ThenFunc(a.findByPosition)).Queries(
```